bndtools.core/src/bndtools/types/Function.java

```diff
@@ -1,4 +1,4 @@
-package org.bndtools.utils;
+package bndtools.types;
 
```

Comment:
Perhaps this is an old type, but why don't we use org.osgi.util.function.Function? bnd now uses Promises so this type is around.